### ISSUE TITLE
Import/Upgrade layered solution from environment to other - Language issue

### ISSUE THREAD

**Issue by lbamrtn** [OfficeDev/Project-Accelerator#91]
Hello, 
I have project accelerator solution in 2 environments (with project service core and accelerator installed). 
- Environment DEV config in English.  
- Environment TEST config in English.  

I added French language in my environments.  Downloaded translation file and imported French translation for my fields in both environment. 

When trying to update solution in my test environment I have this error message : "Entity 3ed6cfb0-d3f6-4673-b071-ec3f9c9a81b9 is not a renamable entity. Hence Entity Display Collection Name cannot be modified."
Error code from my log : 0x8004023B

Entity refer to my project table, but I did not modify existing field, just added new ones. 

Can someone help me :)

**Comment by Azzapatazza** [comment]
I am also having this issue but not because of Languages, previously the entity was able to be renamed and I renamed it to Projects from the new Plans name, since this I cannot import my unmanaged solution into a Production environment as the Prod environment expects the entity to be called Plans but I had previously renamed it to projects and now the display name is locked via the service core solution.

**Comment by SSDK-Hello** [comment]
I can confirm this is the error I got as a result of the Project table being named Plan in new environments. I suspect it is nothing todo with languages and that one of the environments has a mismatch.

MSFT Support suggested:

```
We would like to update you we have verified the issue internally and we have found that there is an active layer present in the target environment for “msdyn_project” entity.
Could you please remove the active layer and verify the issue? 
```

But I still didnt manage to resolve this

**Comment by Dlutchy** [comment]
I also have this issue. Its on Table msdyn_project.
`Entity 861ba2a2-6fb9-4e11-a2bf-f1f911c53a44 is not a renamable entity. Hence Entity Display Name cannot be modified`
 I am only using the English Language (So should be no issue there).

Anyone have any updates on fixes for this?

**Comment by j9mchn** [comment]
> I also have this issue. Its on Table msdyn_project. `Entity 861ba2a2-6fb9-4e11-a2bf-f1f911c53a44 is not a renamable entity. Hence Entity Display Name cannot be modified` I am only using the English Language (So should be no issue there).
> 
> Anyone have any updates on fixes for this?

Yes I do, presuming that you have your own solution to work from.  You need to remove the metadata from the msdyn_project entity.  Switch to classic view of power apps and below are the steps to remove the metadata

1. Navigate to your solution -> open entities
2. Select project and click on Add Subcomponent
3. unselect the checkbox "include entity metadata"

If its project accelerator you will need to do it via the XML of the package.  Remove these lines from the XML:
 <LocalizedNames>
<LocalizedName description="Project" languagecode="1033" />
</LocalizedNames>
<LocalizedCollectionNames>
<LocalizedCollectionName description="Projects" languagecode="1033" />
</LocalizedCollectionNames>
</LocalizedNames>

**Comment by Dlutchy** [comment]
> Yes I do, presuming that you have your own solution to work from. You need to remove the metadata from the msdyn_project entity. Switch to classic view of power apps and below are the steps to remove the metadata
> 
> 1. Navigate to your solution -> open entities
> 2. Select project and click on Add Subcomponent
> 3. unselect the checkbox "include entity metadata"


Thank you very much @j9mchn. By removing metadata from the table fixed the issue.

**Comment by Dlutchy** [comment]
I also had this problem with the Project Table. Microsoft Support suggested this which seemed to have removed the issue.
------------------------------------------
Based on your situation, I recommend the following solution. It involves making edits to the customizations.xml file in your exported solution package. By removing the affected properties, you should be able to resolve the issue. Please search for the following lines using Ctrl+F with the query "description="Project":

<LocalizedNames>
<LocalizedName description="Project" languagecode="1033" />
</LocalizedNames>
<LocalizedCollectionNames>
<LocalizedCollectionName description="Projects" languagecode="1033" />
</LocalizedCollectionNames>

Once you've removed these lines, the problem should be addressed, you can retry the import.
